pkg/cmd/pulumi/operations/refresh.go

```diff
@@ -328,6 +346,11 @@ func NewRefreshCmd() *cobra.Command {
 	targets = cmd.PersistentFlags().StringArrayP(
 		"target", "t", []string{},
 		"Specify a single resource URN to refresh. Multiple resource can be specified using: --target urn1 --target urn2")
+	excludes = cmd.PersistentFlags().StringArrayP(
+		"exclude", "x", []string{},
```

Comment:
Are we sure we should use the short `x` already for this? Or should we see if there's more demand for that first?